python/lsst/pipe/drivers/coaddDriver.py

```diff
@@ -35,13 +35,19 @@ class CoaddDriverConfig(Config):
                         doc="Run detection on the coaddition product")
     detectCoaddSources = ConfigurableField(
         target=DetectCoaddSourcesTask, doc="Detect sources on coadd")
+    fakesAdded = Field(dtype=bool, default=False, doc="Have fakes been added?")
 
     def setDefaults(self):
         self.makeCoaddTempExp.select.retarget(NullSelectImagesTask)
         self.assembleCoadd.select.retarget(NullSelectImagesTask)
         self.assembleCoadd.doWrite = False
 
     def validate(self):
+        if self.fakesAdded:
```

Comment:
I don't think validate should be used to assign anything itself, it should only be used to check if options were set in a consistent way.